wikipedia_cleanup/random_forest.py

```diff
@@ -0,0 +1,141 @@
+import hashlib
+import pickle
+from datetime import date, datetime, timedelta
+from pathlib import Path
+from typing import List, Tuple
+
+import numpy as np
+import pandas as pd
+from sklearn.ensemble import RandomForestClassifier
+from tqdm.auto import tqdm
+
+from wikipedia_cleanup.predictor import Predictor
+
+
+class RandomForestPredictor(Predictor):
+    def __init__(self, use_cache: bool = True) -> None:
+        super().__init__()
+        self.regressors: dict = {}
+        self.last_preds: dict = {}
+        self.hash_location = Path("cache") / self.__class__.__name__
+        self.use_hash = use_cache
+
+    @staticmethod
+    def get_relevant_ids(identifier: Tuple) -> List[Tuple]:
+        return []
+
+    @staticmethod
+    def get_relevant_attributes() -> List[str]:
+        return [
+            "value_valid_from",
+            "day_of_year",  # values from feature engineering
```

Comment:
Could you maybe write a list in the feature engineering code where you just import the features here? Otherwise some features might get missed if we ever add something again.